src/App.jsx

```diff
@@ -14,70 +16,72 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setselectedGood] = useState('Jam');
+  const reset = () => setselectedGood(null);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood
+          ? `${selectedGood} is selected`
+          : 'No goods selected'
+        }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={reset}
+          />
+        )}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {
+            goods.map(good => (
+              <tr
+                key={good}
+                data-cy="Good"
+                className={cn({
+                  'has-background-info-light': selectedGood === good,
+                })
+                }
+              >
+                <td>
+                  <button
+                    data-cy={selectedGood === good
+                      ? 'RemoveButton'
+                      : 'AddButton'
+                    }
+                    type="button"
+                    className={selectedGood === good
+                      ? 'button is-info'
+                      : 'button'
+                    }
+                    onClick={
+                      selectedGood === good
+                        ? reset
+                        : () => setselectedGood(good)
+                    }
+                  >
+                    {selectedGood === good
+                      ? '-'
+                      : '+'
+                    }
+                  </button>
```

Comment:
if we change almost everything depending on the `selectedGood === good` condition, it's better to use something like this

<img width="960" alt="image" src="https://github.com/mate-academy/react_goods-selector-js/assets/70220667/4ca86acb-7bf8-4fe3-8dcf-995407c6fa49">